### PR TITLE
71 update comment markup

### PR DESCRIPTION
**Pull request by mrengy**
I think this takes care of #71 but please double check! It's using a Walker, which is something I haven't done before. 

Related documentation:

1. https://stackoverflow.com/questions/73346121/how-do-i-restructure-the-comment-html-in-wordpress
2. https://developer.wordpress.org/reference/classes/walker_comment/

### CODE REVIEW

File: wp-content/themes/vanguard-history/functions.php [hunk 1]
```diff
@@ -624,3 +624,136 @@ function vanguard_history_all_media_for_year_story() {
...
+						$comment_author      = get_comment_author( $comment );
+						$avatar              = get_avatar( $comment, $args['avatar_size'] );
+					?>
+					<div class="comment-avatar">
+						<?php
+							if ( 0 != $args['avatar_size'] ) {
+								if ( empty( $comment_author_url ) ) {
+									echo $avatar;
+								} else {
+									printf( '<a href="%s" rel="external nofollow" class="url">', $comment_author_url );
```
Comment (shekharkhedekar) [inline wp-content/themes/vanguard-history/functions.php:659]: Make sure to close this `<a>` tag

Comment (shekharkhedekar) [reply wp-content/themes/vanguard-history/functions.php:659]: Also this link resolves to `http://vanguard-history.local/wp` locally - that seems incorrect?

**Review by shekharkhedekar — CHANGES_REQUESTED** [review]

**Comment by shekharkhedekar** [comment]
Looks very close! Thanks for doing this. I'll rebase https://github.com/mrengy/vanguard-history/pull/73 when you merge this.

File: wp-content/themes/vanguard-history/functions.php (hunk 1 above)
Comment (mrengy) [reply wp-content/themes/vanguard-history/functions.php:659]: Good catch. I think https://github.com/mrengy/vanguard-history/pull/72/commits/7d731daab02cb4f74d2a6ace9d7781863ff0dbde resolves the unclosed A tag. 

As for what the link resolves to, that seems correct, actually. For me it was the same aside from the fact that my site url is different and matched what was set for "website" in my profile at wp-admin/users.php . Note that changing these fields on a user profile won't update old comments retroactively

**Review by shekharkhedekar — APPROVED** [review]
thanks!

**Pull request merged**